analytic_account_sequence/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Luxim d.o.o.
+# Copyright 2017 Matmoz d.o.o.
+# Copyright 2017 Deneroteam.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Analytic account code sequence',
+    'summary': 'Analytic account code sequence',
+    'version': '10.0.1.0.0',
+    'author':   'Eficent, '
+                'SerpentCS ,'
+                'Project Expert Team ,'
+                'Odoo Community Association (OCA)',
+    'website': 'https://www.github.com/OCA/account-analytic',
+    'category': 'Project Management',
+    'depends': ['analytic', 'project', 'account_analytic_parent'],
```

Comment:
project?

Reply:
It depends but it should not, I'll remove all the dependent code.